From 0226ba263019d2d865e06f971f085314547ba9be Mon Sep 17 00:00:00 2001
Message-ID: <0226ba263019d2d865e06f971f085314547ba9be.1699318203.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1699318202.git.sharlatanus@gmail.com>
References: <cover.1699318202.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Tue, 7 Nov 2023 00:12:14 +0000
Subject: [PATCH 14/16] gnu: cl-metabang-bind: Update to 0.8.0-2.0819642.

* gnu/packages/lisp-xyz.scm (sbcl-metabang-bind): Update to 0.8.0-2.0819642.
[source]: Update URL as maintainer and owner of repository was changed.

Change-Id: I5fdde9e53e0b7e9cd585ebda85eebe3c29b1d841
---
 gnu/packages/lisp-xyz.scm | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/gnu/packages/lisp-xyz.scm b/gnu/packages/lisp-xyz.scm
index 456180554a..57f9c363e7 100644
--- a/gnu/packages/lisp-xyz.scm
+++ b/gnu/packages/lisp-xyz.scm
@@ -7726,8 +7726,8 @@ (define-public ecl-burgled-batteries3
   (sbcl-package->ecl-package sbcl-burgled-batteries3))
 
 (define-public sbcl-metabang-bind
-  (let ((commit "c93b7f7e1c18c954c2283efd6a7fdab36746ab5e")
-        (revision "1"))
+  (let ((commit "08196426cb099db0623e6cae2aeca566e0b788b2")
+        (revision "2"))
     (package
       (name "sbcl-metabang-bind")
       (version (git-version "0.8.0" revision commit))
@@ -7735,12 +7735,12 @@ (define-public sbcl-metabang-bind
        (origin
          (method git-fetch)
          (uri (git-reference
-               (url "https://github.com/gwkkwg/metabang-bind")
+               (url "https://github.com/hraban/metabang-bind")
                (commit commit)))
          (file-name (git-file-name name version))
          (sha256
           (base32
-           "0hd0kr91795v77akpbcyqiss9p0p7ypa9dznrllincnmgvsxlmf0"))))
+           "14g7k3zhm8cd6bssc5mm5h6iq1dv5lfhiq33aimcmj5a6vbiq47d"))))
       (build-system asdf-build-system/sbcl)
       (native-inputs
        (list sbcl-lift))
-- 
2.41.0

